src/lib/merkletree/proof.ts

```diff
@@ -10,7 +10,7 @@ import { Bytes } from '../../types';
 export interface ProofJSON {
   existence: boolean;
   siblings: string[];
-  nodeAux: NodeAuxJSON | undefined;
```

Comment:
we can't rename it

Reply:
I agree we can't just rename the field.
I think we do: `nodeAux = nodeAux ? node_aux ? undefined;`